Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreeTransaction.h

```diff
@@ -97,6 +101,9 @@ class RemoteLayerTreeTransaction : public CanMakeCheckedPtr<RemoteLayerTreeTrans
             CustomData, // PlatformCALayerRemoteCustom
 #if ENABLE(MODEL_ELEMENT)
             Ref<WebCore::Model>, // PlatformCALayerRemoteModelHosting
+#if ENABLE(MODEL_PROCESS)
+            Ref<WebCore::HostedModel>, // PlatformCALayerRemoteCustom
```

Comment:
It would be nice not to need both this and the Model.

Reply:
Agreed. I think we need to clean up all of the previous model code (that's before the `MODEL_PROCESS` flag) at some point.